test/auth.test.js

```diff
@@ -0,0 +1,88 @@
+import 'babel-polyfill';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import models from '../src/api/models/index';
+
+import server from '../src/index';
+
+// const should = chai.should();
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('Signup', () => {
+  it('should register and give the token', (done) => {
+    chai.request(server)
+      .post('/api/auth/signup')
+      .send({
+        names: 'Burindi Alain',
+        email: 'alain@gmail.com',
+        password: 'password',
+        confirmPassword: 'password'
+      })
+      .end((err, res) => {
+        res.should.have.status(201);
+        res.should.be.an('Object');
```

Comment:
Nitpick: do we really care if it's an object? I think we should focus more on checking the content of the object returned.